cityofaustin/atd-vz-data
# Data model: crash details: change log

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/17280

This is a bare bones change log for the crash details page. 

## Testing
**URL to test:** Local

I guess the main thing to test is that we are are (1) capturing all changes as expected and (2) make sure that this change log is usable and intuitive enough for MVP purposes.

1. Start the data model stack and run the CRIS import script as described in the [collector branch PR](https://github.com/cityofaustin/atd-vz-data/pull/1440).
2. Navigate to any crash details page and scroll down to find the **Record history** card. This is the change log. Is the name **Record history** ok? Do you prefer **Changes** or **Change log**?
3. Click on a row of type `create` in the **Record history** table. Verify that a modal appears which displays a table with the record's values.
4. Try editing a unit and person record, as well as various crash fields. Each time you make an edit, the record history table will update and capture your changes. If the change you made affects a calculated value, e.g., person injury severity, the changes to calculated fields will also be captured in the change log.



---
#### Ship list
